### PR TITLE
Add dockerfile

### PR DESCRIPTION
**Pull request by j6k4m8**
- [x] Dockerfile
- [ ] Add documentation
- [ ] Add to dockerhub
- [ ] Add one-liner docs to join an ingest job from a fresh machine

### CODE REVIEW

**Comment by lrodri29** [comment]
Interesting... Wondering why the dockerfile? How have you used it? It seems to have very little in it. Just git and ingest-client.

**Comment by j6k4m8** [comment]
Yeah probably still needs work (meant to set this as a Draft PR). Main reasons would be to avoid blosc/etc install on a host machine, and to enable fast provisioning of another node for a cloud ingest job. e.g. I start an ingest job on a machine, then ssh into a few new nodes and can join that with a docker one-liner.

I've used this, but decided to `exec` into the container in order to install and configure other packages. Not sure what the best protocol is for that in "real life." Maybe this just is a one-off solution and we don't need docker in this repo? I'm fine with that too :)

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+FROM python:3.7
+
+RUN apt -y update
+RUN apt -y install git
```
Comment (derek-pryor) [inline Dockerfile]: you want to chain the update and install commands together, else caching can mess up by installing old versions.

`RUN apt -y update && apt -y install git`

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+RUN git clone https://github.com/jhuapl-boss/ingest-client
+RUN pip install ./ingest-client
```
Comment (derek-pryor) [inline Dockerfile]: If you install directly from Github then you don't incur an extra layer that is not needed.

`RUN pip install https://github.com/jhuapl-boss/ingest-client`

Comment (j6k4m8) [reply Dockerfile]: Got it! Will fix this in the next commit. 

Original rationale was that I wanted to have an opportunity for users to mount custom plugin files; but don't think that makes sense for the base image here.

**Pull request merged**